### PR TITLE
Unit testing rock paper scissors delete unneeded

### PR DESCRIPTION
**Pull request by Jahteo**
Deletes unneeded files from rock-paper-scissors exercises:

- duplicate & nested readme's & package.json's
- public files
- test files not being used

### CODE REVIEW

**Comment by chasenlehara** [comment]
Links for testing:


| Exercise                     | Problem                                                                                                                                                       | Solution                                                                                                                                                        |
| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| getStringLength              | [Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/getStringLength)                                |                                                                                                                                                                 |
| Basics-together              | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Basics-together/problem?file=/src/index.test.ts) | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Basics-together/solution?file=/src/index.test.ts) |
| Basics                       | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Basics/problem?file=/src/index.test.ts)          | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Basics/solution?file=/src/index.test.ts)          |
| buildDropdownList            | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/buildDropdownList/problem)                       | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/buildDropdownList/solution)                       |
| Button                       | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Button/problem)                                  | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Button/solution)                                  |
| Simplify-rock-paper-scissors | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Simplify-rock-paper-scissors/problem)            | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/Simplify-rock-paper-scissors/solution)            |
| test-rock-paper-scissors     | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/test-rock-paper-scissors/problem)                | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/test-rock-paper-scissors/solution)                |
| renderHook                   | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/renderHook/problem)                              | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/renderHook/solution)                              |
| Temp - vehicle selector      | [CodeSandbox](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/unit-testing-rock-paper-scissors-delete-unneeded/unit-testing-react-what-why-how/vehicle-selector-react)                      | Runs, but can't run tests... use for multiple exercises?                                                                                                        |

File: unit-testing-react-what-why-how/Simplify-rock-paper-scissors/problem/src/App.test.jsx
Comment (chasenlehara) [inline unit-testing-react-what-why-how/Simplify-rock-paper-scissors/problem/src/App.test.jsx:1]: Double checking: is this file supposed to be deleted? Should there be a `.test.` file somewhere in the `problem`?

**Review by chasenlehara — APPROVED** [review]
Let’s merge it!

File: unit-testing-react-what-why-how/Simplify-rock-paper-scissors/problem/src/App.test.jsx
Comment (Jahteo) [reply unit-testing-react-what-why-how/Simplify-rock-paper-scissors/problem/src/App.test.jsx:1]: Tests for the solution are written in a different exercise, so I removed them from here to focus folks on the code exercise.  
Let me know if you want me to add them back in

**Pull request merged**